src/Sulu/Bundle/AdminBundle/Command/DownloadLanguageCommand.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class DownloadLanguageCommand extends Command
+{
+    const TRANSLATION_BASE_URL = 'https://translations.sulu.io/';
```

Comment:
I thought about putting that into a container parameter, but I think it is not worth it... I don't see any real reason to change it, and even if there would be, you can still override that constant by inheritance, because I've used the `static` keyword.